group1/comm/static/comm_tool.js

```diff
@@ -291,7 +291,7 @@ function add_new_room(room) {
 function populate_user_list() {
   $.getJSON('http://' + server_host + ':' + server_port + '/api/users/?format=json', function(all_users) { 
 
-    $.getJSON('http://' + server_host + ':3000/users', function(connected_users){
+    $.getJSON('http://' + host_name + ':3000/users', function(connected_users){
```

Comment:
@nkavanagh good catch. it looks like i missed this when i merged your changes in.